### ISSUE TITLE
Battery fully charged.

### ISSUE THREAD

**Issue by simonccnomis** [Xinyuan-LilyGO/TTGO-T-Display#55]
Is there a way in code to tell when the battery is fully charged ? I know there is the blue LED but I want to display fully charged on the screen because when in the box I made you can't see the LED.
Thanks
Simon

**Comment by Batlapin** [comment]
Nope. LiIon charging considered done when charging current decreases to a certain value and there is no current sensor onboard. But if partially charged (charging transition from CC to CV) detection is OK for you, the BAT_ADC (IO34) can be used.
